lib/jsdom/browser/file/blob.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+function Blob() {
```

Comment:
For new code, please use ES2015 class syntax and style. (E.g. `const` everywhere possible, `let` when not, never `var`.)

If you're feeling brave, you can try using webidl2js, like we are doing for EventTarget. But that is still very early so it might not be ready yet.